modules/apps/object/object-dynamic-data-mapping/bnd.bnd

```diff
@@ -1,3 +1,4 @@
 Bundle-Name: Liferay Object Dynamic Data Mapping
 Bundle-SymbolicName: com.liferay.object.dynamic.data.mapping
-Bundle-Version: 1.0.0
\ No newline at end of file
+Bundle-Version: 1.0.0
+Web-ContextPath: /object-dynamic-data-mapping
```

Comment:
Non-web modules don't usually have a `Web-ContextPath`, but `-form-field-type` ones apparently can. Not sure if Brian will go for this, but I think it might be safer to follow one of these:
- [layout-dynamic-data-mapping-form-field-type](https://github.com/liferay/liferay-portal/tree/master/modules/apps/layout/layout-dynamic-data-mapping-form-field-type)
- [journal-article-dynamic-data-mapping-form-field-type](https://github.com/liferay/liferay-portal/tree/master/modules/apps/journal/journal-article-dynamic-data-mapping-form-field-type)